data/editor.config

```diff
@@ -328,12 +325,11 @@
         <include field="operator" />
```

Comment:
How do we set the subtypes of recycling containers now? Don't we have to add them to the editor?

Reply:
Let's create an issue for it.

Reply:
Done :)